contracts/script/admin-actions/migrate-to-safe/4.RenounceGovernanceRoles.s.sol

```diff
@@ -0,0 +1,87 @@
+/* solhint-disable no-console */
+/* solhint-disable max-line-length */
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+import { TimelockOperations } from "script/utils/TimelockOperations.s.sol";
+import { TimelockController } from "@openzeppelin/contracts/governance/TimelockController.sol";
+import { AccessControl } from "@openzeppelin/contracts/access/AccessControl.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { Create3 } from "src/deploy/Create3.sol";
+
+/// @title RenounceGovernanceRoles
+/// @notice Generates json files with the timelocked operations to renounce roles from the old timelock
+contract RenounceGovernanceRoles is TimelockOperations {
```

Comment:
the renounce governance roles operation should revert if it's not the old address calling. To avoid the new address calling by mistake